skaffold.yaml

```diff
@@ -1,5 +1,10 @@
 apiVersion: skaffold/v2beta24
 kind: Config
+
+# Remove if you desactivate mTLS
+requires:
+  - path: examples/secrets/skaffold.yaml
+
```

Comment:
What does that mean ? this in not required if we do not use mtls ? Shouldn't we parametrize it regarding the mtls activation then ?